electronicparsers/openmx/parser.py

```diff
@@ -716,6 +731,18 @@ def parse(self, mainfile: str, archive: EntryArchive, logger):
         sec_run.program = Program(
             name='OpenMX', version=str(mainfile_parser.get('program_version'))
         )
+        date_start = mainfile_parser.get('date_start')
+
+        elapsed_time = mainfile_parser.get('elapsed_time')
```

Comment:
We should have some checks that the regex actually returns the reasonable (non None) values (the reason might be corrupted output, or some output differences that the parser is not prepared for). Specifically for the date_start this is not and issue, its printed quite early and not much can go wrong, but the elapsed_time is printed at the end, so there is a reasonable assumption that it is not there if something goes wrong during the calculation. So we should check if `mainfile_parser.get('elapsed_time')` is None and in that case don't save `sec_run.time_run.date_end`.

Reply:
OK, sure. I  added :
```
if elapsed_time is not None:
    date_end = date_start_timestamp + elapsed_time
```
Is it what you meant?

Reply:
Almost  :wink:  `sec_run.time_run.date_end = date_end` should be also guarded in the if.